templates/batch-llm/README.ipynb

```diff
@@ -423,7 +423,7 @@
     "### Monitoring Dataset execution\n",
     "We can use the Ray Dashboard to monitor the Dataset execution. In the Ray Dashboard tab, navigate to the Job page and open the \"Ray Data Overview\" section. Click on the link for the running job, and open the \"Ray Data Overview\" section to view the details of the batch inference execution:\n",
     "\n",
-    "<img src=\"assets/ray-data-jobs.png\" width=900px/>"
+    "<img src=\"assets/ray-data-jobs.png\" width=900px />"
```

Comment:
based on html syntax guide, there needs to be a space between `px` and `/>`, otherwise, it will be parsed into `width="900px/"` (which is how python or go's html parsing will return when parsing the string..)

although, web browsers are often super tolerant on these kind of non-standard writing..